config/nginx/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Builds a webserver container that includes the web app static files
+
+# Dockerfile based on the tutorial here:
+#https://medium.com/thepeaklab/how-to-deploy-a-react-application-to-production-with-docker-multi-stage-builds-4da347f2d681
+
+# with build instructions adjusted to match our project documentation
+
+### STAGE 1: Static web files ###
+
+FROM node as build
+RUN mkdir /usr/src/app
+ENV PATH /usr/src/app/node_modules/.bin:$PATH
+COPY frontend/package.json /usr/src/app/package.json
+
+WORKDIR /usr/src/app 
+RUN npm install --silent
+# RUN npm install react-scripts -g --silent # from tutorial but missing in project docs.
```

Comment:
By "project docs" do you mean the README for /frontend or other docs for this project? Just want to make sure I understand what this is referring to.

Reply:
I was looking at the frontend install instructions in the project's top-level README. But after looking around more I just realized that react-scripts is included in the dependencies in package.json . 

I should take out that commented line from the Dockerfile if you agree it doesn't belong there.

Reply:
Let's chat about this at the next meeting. Just wanted to make sure I understood your comment. 

Having something in the `package.json` might not be enough for documentation purposes. Main thing is I'm not sure if `react-scripts` is only needed when running create-react-app the first time or every time the project is built.